tests/infra/logging_app.py

```diff
@@ -203,11 +203,6 @@ def _verify_tx(
                             raise ValueError(
                                 f"Response with status {rep.status_code} is missing 'retry-after' header"
                             )
-                        sleep_time = 0.5
```

Comment:
Why is this removed?

Reply:
There's already another sleep (line `219`) in this loop. While clients should pause for `retry-after` value returned by the historical query endpoint, in the interest of time, I suggest our e2e tests shouldn't actually wait for that long as it's likely the historical queries will be processed quickly enough?